package-lock.json

```diff
@@ -1472,12 +1436,23 @@
                 "htmlparser2": "^3.9.1",
                 "lodash": "^4.15.0",

```

Comment:
Is this going to be an issue or is npm smart enough that it'll just use the more recent version that everything else requires?

Reply:
The version it installs of lodash is 4.17.14 so it should be ok since the ^ indicates that a higher version is compatible.